test/common.py

```diff
@@ -2343,6 +2387,26 @@ def browser_open(cls, url):
     if not EMTEST_BROWSER:
       logger.info('No EMTEST_BROWSER set. Defaulting to `google-chrome`')
       EMTEST_BROWSER = 'google-chrome'
+
+    # If only the the browser is specified, use the default arguments used in circleci.
+    browser_args = shlex.split(EMTEST_BROWSER)
```

Comment:
I think this reproduces the issue with `EMTEST_BROWSER` from before, where on Windows developers would natively do

```
set EMTEST_BROWSER=c:\path\to\browser.exe
```
to set an environment variable to point to an executable, that would not parse through `shlex.split()`.

See https://github.com/emscripten-core/emscripten/pull/24962/files on how to make the parsing work out natively on both Windows and Linux/macOS.